frontend/packages/operator-lifecycle-manager/src/components/modals/update-strategy-modal.tsx

```diff
@@ -0,0 +1,101 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { K8sKind, k8sPatch, K8sResourceKind, Patch } from '@console/internal/module/k8s';
+import {
+  createModalLauncher,
+  ModalBody,
+  ModalSubmitFooter,
+  ModalTitle,
+} from '@console/internal/components/factory/modal';
+import { withHandlePromise } from '@console/internal/components/utils';
+import {
+  ConfigureUpdateStrategy,
+  getNumberOrPercent,
+} from '@console/internal/components/modals/configure-update-strategy-modal';
+
+export const UpdateStrategyModal = withHandlePromise((props: UpdateStrategyModalProps) => {
+  const {
+    path,
+    resource,
+    resourceKind,
+    title,
+    handlePromise,
+    errorMessage,
+    inProgress,
+    cancel,
+    close,
+  } = props;
+  const getPath = path.replace(/\//g, '.');
+  const [strategyType, setStrategyType] = React.useState(
+    _.get(resource, getPath) || props.defaultValue,
+  );
+  const [maxUnavailable, setMaxUnavailable] = React.useState(
+    _.get(resource.spec, 'updateStrategy.rollingUpdate.maxUnavailable', '25%'),
```

Comment:
These paths looks wrong. They should be relative to `path`. See

https://github.com/spadgett/console/blob/dec402f1c240148efe808ccb1f907478a74e14e6/frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx#L528-L549